Seminar3/Task5.py

```diff
@@ -0,0 +1,13 @@
+# Задание №5
+# ✔ Создайте вручную список с повторяющимися целыми числами.
+# ✔ Сформируйте список с порядковыми номерами
+# нечётных элементов исходного списка.
+# ✔ Нумерация начинается с единицы.
+
+my_list = [1, 1, 4, 4, 3, 3, 7, 8, 9]
+new_list = []
+for i, item in enumerate(my_list, start=1):
+    if not item % 2 == 0:    
```

Comment:
Не усложняйте
```suggestion
    if item % 2 != 0:
```
или
```suggestion
    if item % 2:    
```